google/cloud/storage/_http.py

```diff
@@ -53,11 +53,6 @@ def __init__(self, client, client_info=None, api_endpoint=None):
         self.ALLOW_AUTO_SWITCH_TO_MTLS_URL = api_endpoint is None
         self._client_info.client_library_version = __version__
 
-        # TODO: When metrics all use gccl, this should be removed #9552
-        if self._client_info.user_agent is None:  # pragma: no branch
-            self._client_info.user_agent = ""
-        self._client_info.user_agent += " gcloud-python/{} ".format(__version__)
```

Comment:
When this code is removed, will the `user-agent` header be blank?

Reply:
Hi @parthea, this change was made because tracking does not support x-goog-api-client header yet. If we were to remove this now, we would now see version numbers internally.